packages/worker/src/mcp/capabilities/repo/repo-workflow.node.test.ts

```
@@ -264,9 +258,8 @@ test('repo_run_commands reuses resolved target metadata when resuming an existin
 			source_id: 'source-package-1',
 			source_root: '/',
 			base_commit: 'commit-package-1',
-			session_repo_id: 'session-repo-1',
-			session_repo_name: 'repo-package-1-session-1',
-			session_repo_namespace: 'default',
+			session_branch: 'sessions/session-1',
```

Biome (2.5.0)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)